vagrant/Vagrantfile

```diff
@@ -90,6 +90,14 @@ Vagrant.configure("2") do |config|
     v.management_network_address = "192.168.123.0/27"
     v.management_network_name = "qat-mgmt-net"
     v.random_hostname = true
+    # Intel Corporation QuickAssist Technology
+    qat_devices = `lspci | grep -e Quick -e QAT | awk '{print $1}'`
```

Comment:
`grep -e Quick -e QAT` is not reliable. It may not find the cards if the PCI IDs are not updated (which is the case e.g., on my system) or it may find things that are not "supported" by the driver release.

I'd suggest to be explicit about the the devices

```
DEVICES="0434 0435 37c8 6f54 19e2"
qat_devices = `for i in $DEVICES; do lspci -d 8086:$i -m; done|awk '{print $1}'`
```

Also, before doing pass-through, you need to ensure the device is not bound to any driver.

Reply:
@mythi in our initial tests we detected that those devices which were attached by another VM were ignored if those instructions are executed. In the other hand, it should be great to attach one device per VM, did you know if there is a way to detect which devices are not bound?

Reply:
@mythi in our initial tests we detected that those devices which were attached by another VM were ignored if those instructions are executed. In the other hand, it should be great to attach one device per VM, did you know if there is a way to detect which devices are not bound?

Reply:
It's probably that `/sys/bus/pci/devices/0000:<BDF>/driver` does not exist.